Sources/UIAlertControllerView.swift

```diff
@@ -12,6 +12,7 @@ private let minWidth: CGFloat = 300
 
 class UIAlertControllerView: UIView {
     let header = UILabel(frame: .zero)
+    let text = UITextView(frame: .zero)
```

Comment:
We won't need this view if we have no `message`.
It might be better to declare this as an optional variable here and init it only If a `message` is available.
```
let text: UITextView?

// init

if let message = message {
    text = UITextView(frame: .zero)
    text.text = message
    text.font = UIFont.systemFont(ofSize: 14)
   // etc.
}
```